Header header

float32 steering_wheel_angle #in degrees
float32 steering_wheel_torque #in Newton/meters (N/m)

bool enabled
bool override
bool timeout
